modules/microvm/virtualization/microvm/guivm.nix

```diff
@@ -186,10 +192,6 @@ let
   cfg = config.ghaf.virtualization.microvm.guivm;
   vsockproxy = pkgs.callPackage ../../../../packages/vsockproxy { };
 
-  # Importing kernel builder function and building guest_graphics_hardened_kernel
```

Comment:
Let's not remove these but discuss on how we maintain the hardened kernel configuration